### PR TITLE
Fix for issue #6498

### PR DESCRIPTION
**Pull request by praries880**
Fix for issue #6498

Update SimpleParameterSet for New-AzureRmVmss to set SinglePlacementGroup to false by default and add switch parameter `SinglePlacementGroup` that enables the user to create the VMSS in a single placement group.

### CODE REVIEW

**Comment by praries880** [comment]
@hyonholee  @cormacpayne  kindly review the PR.

**Comment by hyonholee** [comment]
LGTM.  Thanks.

File: src/ResourceManager/Compute/Commands.Compute/Manual/VirtualMachineScaleSetCreateOrUpdateMethod.cs [hunk 1]
```diff
@@ -124,6 +124,9 @@ public partial class NewAzureRmVmss : ComputeAutomationBaseCmdlet
         [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
         public int[] DataDiskSizeInGb { get; set; }
 
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false, HelpMessage ="Use this to create the Scale set in a single placement group, default is multiple groups")]
+        public SwitchParameter SinglePlacementGroup;
```
Comment (cormacpayne) [inline src/ResourceManager/Compute/Commands.Compute/Manual/VirtualMachineScaleSetCreateOrUpdateMethod.cs]: @praries880 we need to regenerate the markdown help for this cmdlet to reflect the new parameter

Comment (praries880) [reply src/ResourceManager/Compute/Commands.Compute/Manual/VirtualMachineScaleSetCreateOrUpdateMethod.cs]: done

**Comment by praries880** [comment]
on demand : https://azuresdkci.westus2.cloudapp.azure.com/job/powershell-demand/986/

**Pull request merged**